docs/meson.build

```diff
@@ -0,0 +1,19 @@
+project('libasync_docs')
+
+hotdoc = import('hotdoc')
+docs = hotdoc.generate_doc('libasync',
+	project_version: '0',
+	sitemap: 'sitemap',
+	index: 'src/index.md',
+	extra_assets: [ 'assets/' ],
+)
+
+
+custom_target('libasync-doc-always',
+	build_by_default: true,
+	build_always_stale: true,
+	output: 'phony',
```

Comment:
I'd rather have a `run_target` without the `phony` hack (even if that means invoking it manually).